afs-ws/afs-ws-server-utils/src/test/java/com/powsybl/afs/ws/server/utils/CodeCoverageTest.java

```diff
@@ -16,5 +20,15 @@
     @Test
     public void testNothing() {
         // TODO: remove this class once a real test has been added
```

Comment:
I guess we can remove this todo :) 

At least we should move the tests to another test method.

Reply:
well.. the content of this method still really test nothing...